homeassistant/helpers/integration_platform.py

```diff
@@ -31,7 +31,7 @@ class IntegrationPlatform:
     """An integration platform."""
 
     platform_name: str
-    process_job: HassJob
+    process_job: HassJob[[HomeAssistant, str, Any], Awaitable[None] | None]
```

Comment:
🤦 for forgetting to add this

Reply:
That shouldn't happen after a few more updates. I'm really just waiting for the next Mypy release. So many new things that will be possible. I really hope it gets released next week.